dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -874,8 +874,7 @@ def assert_caching_enabled(cache_control_header, max_age, proxy_max_age)
     sign_in @student
 
     get :show, script_id: sl.script, stage_id: sl.stage, id: sl, solution: true
-
-    assert_response :forbidden
+    assert_select '.teacher-panel', 0
```

Comment:
Talked over this part w/ Ryan - we agreed that for both students and teachers on plc, it's better to show them the level without solution than a forbidden page